frontend/controller/utils/misc.js

```diff
@@ -1,5 +1,20 @@
 'use strict'
 
+import L from '../../common/translations.js'
+
+export const GLOBAL_DASHBOARD_SETTINGS: {[string]: Object } = {
+  'news-and-updates': {
+    title: L('News & Updates'),
+    routeTo: '/global-dashboard/news-and-updates',
+    icon: 'newspaper'
+  },
+  'direct-messages': {
+    title: L('Direct Messages'),
+    routeTo: '/global-dashboard/direct-messages',
+    icon: 'comment'
+  }
+}
```

Comment:
To me this feels a bit out of place. Perhaps it would make more sense to store this route-related information in `frontend/controller/router.js`?

Reply:
@taoeffect sure!

Reply:
@taoeffect I tried to move it to `router.js` and for some reason the entire app started to crash after this change is made, producing below error message. 

![image](https://github.com/okTurtles/group-income/assets/17641213/418913b3-2a8e-4d43-9faa-02721bee463e)

Couldn't really find the clear reason for this during my couple of hours of investigation, but it **seems** to be related to some sort of a circular structure created like below as a result of this movement.

1) `GlobalDashboard.vue` imports something from 'router.js'
2) router.js imports `GlobalDashboard.vue` again. (`GlobalDashboard.vue` is a page component so `router.js` needs to do so to register a route for it).
3) something gets messed up while Vue initialisation process.

So I decided to move it back to the `misc.js`.

Reply:
@SebinSong in that case, can you move it into `GlobalDashboard.vue`? That seems more relevant than `misc.js`

Reply:
@taoeffect Updated!